OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -47,11 +45,11 @@ public Move(Actor self, CPos destination)
 		{
 			mobile = self.Trait<Mobile>();
 
-			getPath = () =>
+			getPath = (check) =>
```

Comment:
Style nit: omit the `()` when there's only one argument.

Reply:
Fixed.